src/AutomaticMechanismAnalysis.jl

```diff
@@ -273,7 +273,7 @@ function analyzespc(sim,spcname,t;N=10,tol=1e-3,branchthreshold=0.9,
     end
     push!(clusternames,"Stables")
 
-    @views dSdtspc = dSdt[length(sim.domain.phase.species)+sim.domain.parameterindexes[1]:sim.domain.parameterindexes[2]]
+    @views dSdtspc = dSdt[length(sim.domain.phase.species)+1:end]
```

Comment:
reverse this change

Reply:
This change is actually correct, because the dSdt returned from transitory sensitivities here, is in terms of the individual domain, not the overall system simulation.

Reply:
It should be the reaction part, "spc" is intended to indicate this is the transitory sensitivity of a particular species rather than sensitivity to a particular species thermochemistry.

Reply:
I've renamed to dSdtrxn

Reply:
I see. Thanks!

Reply:
Can we squash the rename commit here too

Reply:
The dSdt is domain specific